downstream/modules/hub/proc-add-container-readme.adoc

```diff
@@ -27,7 +27,7 @@ Add a README to your container repository to provide instructions to your users
 
 . Navigate to menu:Execution Environments[].
 . Select your container repository.
-. On the *Detail* tab, click btn:[Add].
+. On the *Details* tab, click btn:[Add].
```

Comment:
Hey @EMcWhinn , On the Automation Hub UI, it is the Detail tab.  See screen the capture attached and let me know if you have questions:
![Screenshot from 2023-09-11 10-00-51](https://github.com/ansible/aap-docs/assets/76975926/b483b975-5c1b-4d49-8885-6414e5097542)